i18n/de/middleware/README.md

```diff
@@ -632,6 +632,13 @@ Im Kapitel [Template Beispiele](#template-beispiele) finden Sie ein Template, da
 Eine Schritt-für-Schritt Anleitung für diese Konfiguration kann [hier](step-by-step/3-hardware-tse-at-the-main-cash-register-for-several-additional-cash-registers/README.md) nachgelesen werden.
 
 ### Eine Cloud-TSE für mehrere Kassen
+
+:::caution
+
+Diese Setup Szenario erfordert eine **fiskaly TSE als Teil eines TSE-as-a-Service Pakets**. Weder eine Swissbit Cloud TSE, noch ein fiskaly TSE Einzelprodukt wäre in diesem Anwendungsfall unterstützt.
```

Comment:
We could add here that if people want to use a Single Swissbit Cloud TSE with multiple Queues, they should use a the "1 SCU <-> n Queues" approach

Reply:
True, though, that should also work with a standalone fiskaly SCU, right? In that case the whole sentence should be rephrased.